tools/versions.md

```diff
@@ -21,118 +21,123 @@ How to read the columns:
 
 |Package|Version|Status|Ubuntu packages|OSRF packages|ROS packages|Gazebo Classic|Gazebo|Release date|EOL date|
 |---|---|---|---|---|---|---|---|---|---|
-|gz-cmake|0| stable |B C D E F|X Z A B||||2017-10-09|2023-01-25 with Gazebo 9|
+|gz-cmake|0| EOL ❌ |B C D E F|X Z A B||||2017-10-09|2023-01-25 with Gazebo 9|
 ||1|EOL ❌||X B||||2018-12-12|-|
-||2|stable|F G H I|B|F|11|A B C D E F G|2019-01-31|TBD|
-||3|not-released||||||TBD|TBD|
+||2|stable|F|B F J|F J|11|A B C D E F|2019-01-31|2026-09 with Fortress|
+||3|stable||F J|||G|2022-09-23|TBD|
 |gz-common|0|EOL ❌||T X Y Z A||||2016-07-27|-|
-||1|stable|B C D E F|X Z A B||9 10||2018-01-05|2023-01-25 with Gazebo 9|
+||1|EOL ❌|B C D E F|X Z A B||9 10||2018-01-05|2023-01-25 with Gazebo 9|
 ||2|EOL ❌||X B||||2019-02-11|-|
-||3|stable|G H I|B F|F|11|A B C D|2019-02-28|2025-01-25 with Gazebo 11|
-||4|stable|||F||E F|2020-03-31|2026-09 with Fortress|
-||5|not-released|||||G|2022-09|TBD|
+||3|stable|G H I J K L|B F|F|11|A B C D|2019-02-28|2025-01-25 with Gazebo 11|
+||4|stable|J K L|B F J|F J| |E F|2020-03-31|2026-09 with Fortress|
+||5|stable||F J|||G|2022-09-22|TBD|
 |gz-fuel-tools|0|EOL ❌||X Z A||||2017|-|
-||1|stable|B C D E F|X A B||9 10||2018-01-25|2023-01-25 with Gazebo 9|
+||1|EOL ❌|B C D E F|X A B||9 10||2018-01-25|2023-01-25 with Gazebo 9|
 ||2|EOL ❌||X B||||2019-01-29|2019-01-29|
 ||3|EOL ❌||B|||A B|2019-01-30|2020-12 with Blueprint|
-||4|stable|G H I|B F|F|11|C|2019-12-10|2025-01-25 with Gazebo 11|
+||4|stable|G H I J K L|B F|F|11|C|2019-12-10|2025-01-25 with Gazebo 11|
 ||5|EOL ❌|||||D|2020-09-30|2021-12 with Dome|
 ||6|EOL ❌|||F||E|2020-03-31|2022-03 with Edifice|
-||7|stable|||||F|2021-09-30|2026-09 with Fortress|
-||8|not-released|||||G|2022-09|TBD|
+||7|stable|J K L|B F J|F J||F|2021-09-30|2026-09 with Fortress|
+||8|stable||F J|||G|2022-09-26|TBD|
 |gz-sim|1|EOL ❌||B|||A|2019-03-02|2019-09 with Acropolis|
 ||2|EOL ❌||B|||B|2019-05-21|2020-12 with Blueprint|
 ||3|stable||B F|F||C|2019-12-10|2024-12 with Citadel|
 ||4|EOL ❌|||||D|2020-09-30|2021-12 with Dome|
 ||5|EOL ❌|||F||E|2020-03-31|2022-03 with Edifice|
-||6|stable|||||F|2021-09-30|2026-09 with Fortress|
-||7|not-released|||||G|2022-09|TBD|
-|gz-gui<br>|0|EOL ❌||B||||2019-03-06|-|
+||6|stable|J K L|B F J|F J||F|2021-09-30|2026-09 with Fortress|
+||7|stable||F J|||G|2022-09-27|2024-11 with Garden|
+||8|not-released|||||H|2023-09|TBD|
+|gz-gui|0|EOL ❌||B||||2019-03-06|-|
 ||1|EOL ❌||B|||A|2019-03-01|2019-09 with Acropolis|
 ||2|EOL ❌||B|||B|2019-05-21|2020-12 with Blueprint|
 ||3|stable||B F|F||C|2019-12-10|2024-12 with Citadel|
 ||4|EOL ❌|||||D|2020-09-30|2021-12 with Dome|
 ||5|EOL ❌|||F||E|2020-03-31|2022-03 with Edifice|
-||6|stable|||||F|2021-09-30|2026-09 with Fortress|
-||7|not-released|||||G|2022-09|TBD|
+||6|stable|J K L|B F J|F J||F|2021-09-30|2026-09 with Fortress|
+||7|stable||F J|||G|2022-09-27|TBD|
```

Comment:
add gui8?

```
||8|not-released|||||H|2023-09|TBD|
```

and change 7's EOL to `2024-11 with Garden`

Reply:
Done in [1c1513e](https://github.com/gazebosim/docs/pull/335/commits/1c1513ead357d9090cb764ded6a6be653a7596f7)